src/java/org/apache/cassandra/db/partitions/TriePartitionUpdate.java

```diff
@@ -300,6 +303,18 @@ public int operationCount()
              + (deletionInfo().getPartitionDeletion().isLive() ? 0 : 1);
     }
 
+    /**
+     * The accumulated BTree size of the data contained in this update.
+     *
+     * @return the accumulated BTree size of the data contained in this update.
+     */
+    @VisibleForTesting
+    @Override
+    public long accumulatedDataSize()
+    {
+        return dataSize;
+    }
```

Comment:
For Review:
`accumulatedDataSize` and `unsharedHeapSize` are coming from 5.0 before `PartitionUpdate` became an interface. The implementation has moved to `BTreePartitionUpdate` and `TriePartitionUpdate`.